import catchAsyncError from '../../../errors/catchAsyncError.js';
import ErrorHandler from '../../../errors/errorHandler.js';
import models from '../../../models/index.js';


/// LOGIN USER ///

const login = catchAsyncError(async (req, res, next) => {})
export default login;